action.yml

```diff
@@ -1,7 +1,6 @@
 name: Changesets
 description: A GitHub action to automate releases with Changesets
 runs:
-  using: "node16"
```

Comment:
what exact impact that would have on consumers of this action though? isn't it recommended to use an explicit version here?